misk/src/test/kotlin/misk/metrics/web/MetricsJsonActionTest.kt

```diff
@@ -3,15 +3,18 @@ package misk.metrics.web
 import com.google.common.truth.Truth.assertThat
 import misk.metrics.Metrics
 import misk.metrics.MetricsModule
-import misk.testing.MiskTestRule
-import org.junit.Rule
-import org.junit.Test
+import misk.testing.MiskTest
+import misk.testing.ModuleProvider
+import misk.testing.Modules
+import org.junit.jupiter.api.Test
 import java.util.concurrent.TimeUnit
 import javax.inject.Inject
 
+@MiskTest
 internal class MetricsJsonActionTest {
-    @get:Rule
-    val miskTestRule = MiskTestRule(MetricsModule())
+    @Modules val modules = ModuleProvider(
```

Comment:
... in which case this wouldn’t need a ModuleProvider at all

Reply:
... yup.

Reply:
Done in this PR since I had to rebase anyway.